x/denommetadata/client/cli/utils.go

```diff
@@ -0,0 +1,22 @@
+package cli
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/osmosis-labs/osmosis/v15/osmoutils"
+	"github.com/spf13/cobra"
+)
+
+func parseProposal(cmd *cobra.Command) (osmoutils.Proposal, sdk.Coins, error) {
```

Comment:
you already have it in dymension/utils. why creating a new utils for parseProposal?

Reply:
removed